package.json

```diff
@@ -6,9 +6,11 @@
         "node": ">=8"
     },
     "scripts": {
-        "start": "react-scripts start",
+        "start": "npm-run-all -p watch:css start:app build",
```

Comment:
Small thing, but the `--parallel` option might be more clear.

From [Why are there short and long alternatives for command line options?](https://superuser.com/a/174568):

> [long options] were introduced because they are more readable and often more memorable... Many programs now have both: short options for when you're typing on the command line and remember the character, long options for scripts or on the command line if you only remember the longer option name.

Reply:
I don't think we need the build task to be run here. That's just for production.